packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3480,6 +3511,15 @@ class InputDecoration {
   /// a default height based on text size.
   final BoxConstraints? constraints;
 
+  /// Widget that allows to customize the error text.
```

Comment:
Nit: I think we can remove this first line, since this does not customize `errorText`.